plugins/imagebase/plugin.js

```diff
@@ -186,17 +186,290 @@
 		};
 	}
 
+	function getUploadFeature() {
+		/**
+		 * Widget feature dedicated for handling seamless file uploads.
+		 *
+		 * This type serves solely as a mixing, and should be added using
```

Comment:
As for `@since` tag, I don't see much sense for that here as `CKEDITOR.plugins.imagebase` namespace is added in `4.9.0` so it seems natural to me that this info is not needed.

I'll report a feature request in Umberto to inherit/"cascade" since tag.

Reply:
FYI: reported as cksource/umberto#466